config/services_tests_config_uat.json

```diff
@@ -3,21 +3,22 @@
   "batchee": "sambah",
   "geoloco": "geoloco",
   "giovanni-adapter": "TBD",
+  "harmony": "harmony",
   "harmony-gdal-adapter": "hga",
   "harmony-netcdf-to-zarr": "n2z",
   "harmony-regridder": "regridder",
   "harmony-service-example": "harmony-regression",
   "hoss": "hoss, variable-subsetter",
   "hybig": "hybig,opera-rtc-s1-browse",
   "net2cog": "net2cog",
-  "podaac-concise": "TBD",
-  "podaac-l2-subsetter": "TBD",
+  "opera-rtc-s1-browse": "opera-rtc-s1-browse",
+  "podaac-concise": "sambah",
+  "podaac-l2-subsetter": "sambah",
   "query-cmr": "variable-subsetter",
+  "sds-maskfill": "hoss",
```

Comment:
Thanks for updating this, @flamingbear.

For folks who weren't on our Slack call (so everyone else!): MaskFill is used as part of the HOSS service chain, so when the MaskFill Docker image is updated in a Harmony environment, the HOSS regression test suite should run.

Similarly: SAMBAH is a service chain using Batchee, Stitchee, L2SS and Concise. So when the L2SS or Concise Docker images are updated, the SAMBAH regression tests should be invoked.